app/shared/constants/endpoints.js

```diff
@@ -12,4 +12,4 @@
 
 export const UA_SERVICE_PORT = 9090;
 export const UA_SERVICE_HTTP = `http://localhost:${UA_SERVICE_PORT}/v1`;
-export const UA_SERVICE_WS = `ws://localhost:${UA_SERVICE_PORT}/v1`;
+export const UA_SERVICE_WS = `ws://localhost:${UA_SERVICE_PORT}/v1/ws`;
```

Comment:
I would guess that this isn't doing the right thing in the UI client module, since `/diffs` would have had to get pushed in there somewhere.